### DISCUSSION TITLE
All clips in one stream in Home Assistant media browser possible?

### DISCUSSION THREAD

**Discussion by rhatguy** [blakeblackshear/frigate#423] [Ideas]
Would it be a useful feature or would it even be possible to create a stream in the media browser in Home Assistant that was just a single stream for a particular camera for a particular day ?  I.E. instead of having 60 separate clips for a camera, present one long clip of all motion events for the entire day.  I'm wondering if this would be possible to generate on the fly "virtually" if a user requested to play it by "combining" or just streaming the individual clips one after another.  Not sure how that would work with streaming since I'm sure people would want to "fast-forward" within this type of video quite often.

Not a required feature, just an idea.

**Comment by blakeblackshear** [comment]
There have been a few ideas for post processors like this. Here is the existing issue for a recap: https://github.com/blakeblackshear/frigate/issues/54

**Reply by rhatguy** [reply]
Wow...never seen anything like that superimposed video before.  Thats pretty wild.  I'm thinking of something considerably less complex than that.  Basically just creating a digest video on the fly by "concatenating" clips.  Thinking about maintaining a "mylist.txt" (file list for ffmpeg input from https://trac.ffmpeg.org/wiki/Concatenate) for each camera per day, or either generating those on the fly also.  Then when media browser asked for it, fire up an ffmpeg process to stream the resulting video.  That would be very low overhead (capacity and performance) to maintain the list files, so the only added overhead would be ffmpeg concatenating the videos upon request.  I'm probably oversimplifying.  Just throwing ideas out there.

Thanks for all your AWESOME work on this!

**Comment by walthowd** [comment]
VLC also supports "Open Directory" which works really well for the 24x7 one minute files -- it works fairly well for clips, but default behaviour is to play oldest clip per alphabetical sorted camera name first.
